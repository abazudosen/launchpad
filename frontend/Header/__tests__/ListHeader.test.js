/* @flow */

import React from 'react';
import ListHeader from '../ListHeader.js';
import ReactShallowRenderer from 'react-test-renderer/shallow';

const renderer = new ReactShallowRenderer();

test('renders normal', () => {
  const component = renderer.render(
    <ListHeader
      user={{
        id: 'test-id',
        githubUsername: 'example',
        pads: [],
      }}
      onLogout={() => {}}
    />,
  );
  expect(component).toMatchSnapshot();
});
