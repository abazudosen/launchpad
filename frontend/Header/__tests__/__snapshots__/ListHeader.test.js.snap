// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Header renders normal 1`] = `
<div
  className="Header"
>
  <div
    className="Header-Left"
  >
    <HeaderLogo />
    <div
      className="Management-Buttons"
    >
      <HeaderButton
        onClick={[Function]}
      >
        New
      </HeaderButton>
    </div>
  </div>
  <div
    className="Header-Gap"
  />
  <div
    className="Header-Right"
  >
    <UserBar
      onLogin={[Function]}
      onLogout={[Function]}
      user={
        Object {
          "githubUsername": "example",
          "id": "test-id",
          "pads": Array [],
        }
      }
    />
  </div>
</div>
`;
